README.md

```diff
@@ -1,471 +1,187 @@
 
----
+Finally, to shutdown the applications and stop the containers, run:
 
-## Docker
-
-### The Docker Machine
-The default `docker-machine` in a Windows or Mac OS X environment is 1 CPU and 1GB RAM, this is not enough to run the Mauro Data Mapper system.
-On Linux the docker machine is the host machine so there is no need to build or remove anything.
-
-#### Native Docker
+```bash
+docker compose down
+```
 
-If using the Native Docker then edit the preferences of the Docker application and increase the RAM to at least 4GB,
-you will probably need to restart Docker after doing this.
+# Updating
 
-#### Docker Toolbox
+The `Dockerfile` for the `mauro-data-mapper` container image fetches the Mauro commits/snapshots from git. By default, the `develop` branch of each required repository/snapshot is used during the build, so that the latest changes can be incorporated and run.
 
-If using the Docker Toolbox then as such you will need to perform the following in a 'docker' terminal.
+To update the running instance:
 
 ```bash
-# Stop the default docker machine
-$ docker-machine stop default
+# Stop/shutdown any running application
+docker compose down
 
-# Remove the default machine
-$ docker-machine rm default
+# Rebuild the container images with the latest snapshots
+docker compose build
 
-# Replace with a more powerful machine (4096 is the minimum recommended RAM, if you can give it more then do so)
-$ docker-machine create --driver virtualbox --virtualbox-cpu-count "-1" --virtualbox-memory "4096" default
+# Start the containers again
+docker compose up -d
 ```
 
-##### Configuring shell to use the default Docker Machine
+# Volume and Files
 
-When controlling using Docker Machine via your terminal shell it is useful to set the `default` docker machine.
-Type the following at the command line, or add it to the appropriate bash profile file:
+The following shared volumes will be created and used:
 
-```bash
-eval "$(docker-machine env default)"
-```
+- `postgres12` - For storing the postgresql database
+- `lucene_index` - For storing the Lucense search index used by Mauro.
 
-If not you may see the following error: `Cannot connect to the Docker daemon. Is the docker daemon running on this host?`
+Stopping the containers will not remove these volumes, they will persist as storage which the containers connect to.
 
+To view the log file generated by Mauro, it is accessible via:
 
-### Cleaning up docker
+- Source (host): `shared_volumes/logs/maurodatamapper/mauro-data-mapper/mauro-data-mapper.log`
+- Destination (container): `/usr/local/tomcat/logs/mauro-data-mapper/mauro-data-mapper.log`
 
-Continually building docker images will leave a lot of loose snapshot images floating around, occasionally make use of:
+# Cleanup
+
+Continually building docker images will leave a lot of loose snapshot images floating around. Use the following commands to cleanup dangling resources: 
 
 * Clean up stopped containers - `docker rm $(docker ps -a -q)`
```

Comment:
This shell syntax will work in a bash prompt but not in a Windows command prompt

Reply:
Strange, seems to work in PowerShell